src/aggregators/mux/MuxAdapter.sol

```diff
@@ -117,9 +117,11 @@ contract MuxAdapter is Storage, Config, ImplementationGuard, ReentrancyGuardUpgr
         if((flags & POSITION_OPEN) != 0){
             //We will not have to deposit or give the approvals for close position
             if (_account.collateralToken == _WETH) {
-                IWETH(_WETH).deposit{ value: collateralAmount }();
+                require(msg.value >= collateralAmount, "Insufficient ETH");
```

Comment:
bro why >= and not ==?

Reply:
just in case the user / front end sends a little bit more than required, we don't want to cancel the transaction.
We should be able to withdraw it whenever we want